Userland/Libraries/LibGfx/ImageFormats/BMPLoader.cpp

```diff
@@ -1292,8 +1292,10 @@ static ErrorOr<void> decode_bmp_pixel_data(BMPLoadingContext& context)
                 u8 mask = 8;
                 while (column < width && mask > 0) {
                     mask -= 1;
-                    auto color_idx = (byte >> mask) & 0x1;
+                    size_t color_idx = (byte >> mask) & 0x1;
```

Comment:
I'm not sure to see the value of explicitly using a `size_t` here (and 15 lines later)

Reply:
Because the type of `color_idx` is `int` when `auto` is used, the line below gives the following compiler error:

```
/home/tim/repos/serenity/Userland/Libraries/LibGfx/ImageFormats/BMPLoader.cpp:1297:39: error: comparison of integers of different signs: 'int' and 'size_t' (aka 'unsigned long') [-Werror,-Wsign-compare]
    if (color_idx >= context.color_table.size())
        ~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Oh sure, I forgot that we have this warning